code/game/antagonist/outer/mercenary/mercenary.dm

```diff
@@ -52,6 +52,9 @@
 	for(var/name in stat_modifiers)
 		L.stats.changeStat(name, stat_modifiers[name])
 
+	for(var/perk in perks)
```

Comment:
I'd give this to base antagonist equip()

Reply:
Yes, this new code should be in base antag equipd()

Reply:
Do you see a base antagonist equip()?
Base antagonist equip() don't have even stat_modifiers

Reply:
What if it's blitzshell or borer?

Reply:
I resolve this review, because of the build code.

Reply:
They are in code\game\antagonist\antagonist_equip.dm at first line

Reply:
I know where parent proc is, i say that if i do this there will be bunch of runtimes.
There are a reason why stats not in parent proc. What about borer or blitzhell? They don't have stats.
What if somebody want to add their no-human antag? More runtimes.
No, i don't want that.